st2api/st2api/controllers/v1/ruleviews.py

```diff
@@ -85,15 +87,31 @@ class RuleViewController(resource.ContentPackResourceController):
     include_reference = True
 
     def get_all(self, sort=None, offset=0, limit=None, requester_user=None, **raw_filters):
-        rules = self._get_all(sort=sort,
-                              offset=offset,
-                              limit=limit,
-                              raw_filters=raw_filters,
-                              requester_user=requester_user)
+        rules = super(RuleViewController, self)._get_all(sort=sort,
+                                                         offset=offset,
+                                                         limit=limit,
+                                                         raw_filters=raw_filters,
+                                                         requester_user=requester_user)
         result = self._append_view_properties(rules.json)
         rules.json = result
         return rules
 
+    @staticmethod
```

Comment:
It makes sense to me to separate the function into a mixin to reduce repetition.

Reply:
Could I add a new class named `BeseResourceIsolationHandlerMixin`?

Reply:
@cognifloyd  resource_model_filter have been used in ContentPackResourceController which is super class of RuleViewController. So it's no need.